src/graph/delta_matrix/delta_matrix_iter.h

```diff
@@ -35,7 +35,8 @@ GrB_Info Delta_MatrixTupleIter_AttachRange
 	Delta_MatrixTupleIter *iter,    // iterator to update
 	const Delta_Matrix A,           // matrix to scan
 	GrB_Index min_row,              // minimum row for iteration
-	GrB_Index max_row               // maximum row for iteration
+	GrB_Index max_row,              // maximum row for iteration
+	bool transpose                  // iterate over the transpose of A
```

Comment:
I think it would be better to pass in the "correct" matrix and avoid passing this extra boolean.